test/parallel/test-querystring.js

```diff
@@ -304,6 +310,9 @@ assert.strictEqual(
   qs.stringify(obj, null, null, { encodeURIComponent: demoEncode }),
   'a=a&b=b&c=c');
 
+// Test QueryString.unescapeBuffer
+assert.strictEqual(qs.unescape('%C4%97%'), 'ė%');
```

Comment:
I'd prefer having more direct tests against `qs.unescapeBuffer()` in addition to the indirect ones for `qs.unescape()`. [Here](https://github.com/nodejs/node/blob/master/benchmark/querystring/querystring-unescapebuffer.js#L7-L10) are some nice test cases for `unescapeBuffer()`.